### PR TITLE
Revisit 01_project_manipulation

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary
 - Remove too detailed description

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/level-up/basic_skills/01_project_manipulation.rst [hunk 1]
```diff
@@ -146,11 +107,13 @@ Print project information
 
   .. tab-item:: ProjectCLI
 
-    Print an overall information of the project.
+    We can print an overall information of the project.
 
     .. code-block:: bash
 
-      datum project pinfo [-h] [-p PROJECT_DIR] [revision]
+      datum project info
```
Comment (wonjuleee) [inline docs/source/docs/level-up/basic_skills/01_project_manipulation.rst:114]: is this correct? `pinfo`

File: docs/source/docs/level-up/basic_skills/01_project_manipulation.rst [hunk 2]
```diff
@@ -74,20 +56,12 @@ Print dataset information
...
-
-    ``revpath`` is either a dataset path or a revision path. For more information on the revision, please see
-    :ref:`Level 11: Project Versioning`. ``--all`` directive shows all the dataset information of your project.
-
-    For example, the following command is an example of printing information about the :ref:`Cityscapes` dataset from the previous step.
-
-    .. code-block:: bash
-
-      datum project dinfo my-dataset
+      datum dinfo my-dataset
```
Comment (wonjuleee) [inline docs/source/docs/level-up/basic_skills/01_project_manipulation.rst:64]: is there any `dinfo`?

File: docs/source/docs/level-up/basic_skills/01_project_manipulation.rst (hunk 1 above)
Comment (vinnamkim) [reply docs/source/docs/level-up/basic_skills/01_project_manipulation.rst:114]: - https://github.com/openvinotoolkit/datumaro/pull/952 fixed it to `datum project info`.

File: docs/source/docs/level-up/basic_skills/01_project_manipulation.rst (hunk 2 above)
Comment (vinnamkim) [reply docs/source/docs/level-up/basic_skills/01_project_manipulation.rst:64]: - https://github.com/openvinotoolkit/datumaro/pull/910 changed it from `datum info` to `datum project dinfo`.
- https://github.com/openvinotoolkit/datumaro/pull/952 changed it from `datum project dinfo` to `datum dinfo`.

**Review by wonjuleee — APPROVED** [review]

**Review by cih9088 — APPROVED** [review]

**Pull request merged**